src/e2e-test/features/BQExecuter/BQTableCreate.feature

```diff
@@ -0,0 +1,61 @@
+@BQExecute
+Feature: Executes a SQL query in BigQuery Execute
```

Comment:
-> BigQueryExecute - Verify successful data transfer to BQ table with BQExecute plugin

Reply:
@sawantpritam -Done